suse_migration_services/units/prepare.py

```diff
@@ -101,6 +101,24 @@ def main():
     cloud_register_metadata = os.sep.join(
         [root_path, 'var', 'lib', 'cloudregister']
     )
+    zypper_log_path = os.sep.join(
+        [root_path, 'var', 'log', 'zypper.log']
+    )
+    if os.path.exists(zypper_log_path):
```

Comment:
Can we be consistent in the naming, log_file vs. log_path. I vote for zypper_log_file and zypper_host_log_file

Reply:
Fixed